Sorry, I'd made some assumptions from the fact that this didn't build on my machine, and that I saw some deps changes in the upstream changelog.

```
this derivation will be built:
  /nix/store/pkzgdjjnkh8l9n5rll8da18hdd2crmzj-miniflux-2.0.49.drv
building '/nix/store/pkzgdjjnkh8l9n5rll8da18hdd2crmzj-miniflux-2.0.49.drv'...
unpacking sources
unpacking source archive /nix/store/6vfx9rzi9s92nrbfdkjzxp6jc38n0rcq-source
source root is source
patching sources
updateAutotoolsGnuConfigScriptsPhase
configuring
building
Building subPackage .
internal/cli/cleanup_tasks.go:7:2: package log/slog is not in GOROOT (/nix/store/23wrmhyfnwjxc9cvy3lab9xj551ic53c-go-1.20.8/share/go/src/log/slog)
note: imported by a module that requires go 1.21
error: builder for '/nix/store/pkzgdjjnkh8l9n5rll8da18hdd2crmzj-miniflux-2.0.49.drv' failed with exit code 1;
       last 10 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/6vfx9rzi9s92nrbfdkjzxp6jc38n0rcq-source
       > source root is source
       > patching sources
       > updateAutotoolsGnuConfigScriptsPhase
       > configuring
       > building
       > Building subPackage .
       > internal/cli/cleanup_tasks.go:7:2: package log/slog is not in GOROOT (/nix/store/23wrmhyfnwjxc9cvy3lab9xj551ic53c-go-1.20.8/share/go/src/log/slog)
       > note: imported by a module that requires go 1.21
       For full logs, run 'nix-store -l /nix/store/pkzgdjjnkh8l9n5rll8da18hdd2crmzj-miniflux-2.0.49.drv'.
```

--- a/pkgs/servers/miniflux/default.nix
+++ b/pkgs/servers/miniflux/default.nix
@@ -1,20 +1,20 @@
-{ lib, buildGoModule, fetchFromGitHub, installShellFiles, nixosTests }:
+{ lib, buildGo121Module, fetchFromGitHub, installShellFiles, nixosTests }:
 
 let
   pname = "miniflux";
-  version = "2.0.48";
+  version = "2.0.49";
 
-in buildGoModule {
+in buildGo121Module {
   inherit pname version;
 
   src = fetchFromGitHub {
     owner = pname;
     repo = "v2";
     rev = version;
-    sha256 = "sha256-g2Cnkf022aU/kUkb6N8huB+SFY60uNxyI9BVEycl37c=";
+    sha256 = "sha256-MGKQSlpTLqQPmvhACl9fbQkz2Uil8V8btjTwJIcY7g0=";
   };
 
-  vendorHash = "sha256-d4/oDvMRZtetZ7RyCHVnPqA78yPVFyw4UhjfPD1XuMo=";
+  vendorHash = "sha256-J3WHFfmjgE71hK58WP3dq+Px4XxLbluJSGv+eJiIB0E=";
 
   nativeBuildInputs = [ installShellFiles ];
 

